environment/src/main/java/com/sequenceiq/environment/environment/flow/deletion/handler/StorageConsumptionCollectionUnschedulingHandler.java

```diff
@@ -0,0 +1,103 @@
+package com.sequenceiq.environment.environment.flow.deletion.handler;
+
+import static com.sequenceiq.environment.environment.flow.deletion.event.EnvDeleteHandlerSelectors.UNSCHEDULE_STORAGE_CONSUMPTION_COLLECTION_EVENT;
+import static com.sequenceiq.environment.environment.flow.deletion.event.EnvDeleteStateSelectors.START_RDBMS_DELETE_EVENT;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.auth.altus.EntitlementService;
+import com.sequenceiq.environment.environment.dto.EnvironmentDeletionDto;
+import com.sequenceiq.environment.environment.dto.EnvironmentDto;
+import com.sequenceiq.environment.environment.flow.deletion.event.EnvDeleteEvent;
+import com.sequenceiq.environment.environment.service.EnvironmentService;
+import com.sequenceiq.environment.environment.service.consumption.ConsumptionService;
+import com.sequenceiq.flow.reactor.api.event.EventSender;
+import com.sequenceiq.flow.reactor.api.handler.EventSenderAwareHandler;
+
+import reactor.bus.Event;
+
+@Component
+public class StorageConsumptionCollectionUnschedulingHandler extends EventSenderAwareHandler<EnvironmentDeletionDto> {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(StorageConsumptionCollectionUnschedulingHandler.class);
+
+    private final EnvironmentService environmentService;
+
+    private final HandlerExceptionProcessor exceptionProcessor;
```

Comment:
This component is not used in the counterpart `SchedulingHandler`

Reply:
It is only needed for the error handling in the delete flow handlers. The layout / approach is a direct copy paste of other handlers around.